README.md

```diff
@@ -107,31 +103,45 @@ img.scenes
 # Change scene
 img.set_scene("Image:1")
 
-# Same operations on a different scene
-img.dask_data  # returns 5D TCZYX dask array
-img.xarray_dask_data  # returns 5D TCZYX xarray data array backed by dask array
-img.dims  # returns a Dimensions object
-img.dims.order  # returns string "TCZYX"
-img.dims.X  # returns size of X dimension
-img.shape  # returns tuple of dimension sizes in TCZYX order
-img.get_image_dask_data("CZYX", T=0)  # returns 4D CZYX dask array
-
-# Read a specified portion of dask array
-lazy_t0 = img.get_image_dask_data("CZYX", T=0)  # returns 4D CZYX dask array
-t0 = lazy_t0.compute()  # returns 4D CZYX numpy array
-
-# Get a 5D TCZYX dask array
-lazy_data = imread_dask("my_file.tiff")  # optionally provide a scene id, default first
-lazy_t0 = lazy_data[0, :]
-t0 = lazy_t0.compute()
+# Use the same operations on a different scene
+# ...
 ```
 
 #### Delayed Image Reading Notes
 
 The `.dask_data` and `.xarray_dask_data` properties and the `.get_image_dask_data`
-function will not load any piece of the pixel data into memory until you specifically
+function will not load any piece of the imaging data into memory until you specifically
 call `.compute` on the returned Dask array. In doing so, you will only then load the
-selected data in-memory.
+selected chunk in-memory.
+
+### Xarray Coordinate Plane Attachment
+
+If `aicsimageio` finds coordinate information for the spatial-temporal dimensions of
```

Comment:
Along with the words give a concrete conceptual example w/ dims etc.

Reply:
Examples are right below this no?

Reply:
Pinging again @heeler